src/components/Calculator.js

```diff
@@ -1,113 +1,220 @@
+              >
                 6
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-orange">
+              <button
+                type="button"
+                value="-"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-orange"
+              >
                 -
               </button>
             </td>
           </tr>
           <tr>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="1"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 1
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="2"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 2
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="3"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 3
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-orange">
+              <button
+                type="button"
+                value="+"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-orange"
+              >
                 +
               </button>
             </td>
           </tr>
           <tr>
             <td colSpan="2">
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="0"
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 0
               </button>
             </td>
 
             <td>
-              <button type="button" className="btn btn-white">
+              <button
+                type="button"
+                value="."
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-white"
+              >
                 .
               </button>
             </td>
             <td>
-              <button type="button" className="btn btn-orange">
+              <button
+                type="button"
+                value="="
+                onClick={(e) => setObj(calculate(obj, e.target.value))}
+                className="btn btn-orange"
+              >
                 =
```

Comment:
- [ ] Kindly make sure there are no errors in your application, currently, nothing is displayed on the  screen when your app loads, please check your Calculator component and ensure that there are no errors in your console so that onload your application loads and for each button click the onclick function is fired 👍🏿 

### OPTIONAL
- [ ] You can also refactor your click event to a function so that you can reuse it in your app, this will be better to make the function reusable for all the click events